docs/index.md

```diff
@@ -1,11 +1,54 @@
-# Github-runner operator
-A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) deploying self-hosted GitHub runners.
-  
-Each unit of this charm will start a configurable number of LXD based containers and virtual machines to host them. Each runner performs only one
-job, after which it unregisters from GitHub to ensure that each job runs in a clean environment. The charm will periodically check the number of idle runners and spawn or destroy them as
-necessary to maintain the configured number of runners. Both the reconciliation interval and the number of runners to maintain are configurable.
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) deploying and managing [GitHub self-hosted runners](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners) on virtual machines.
+
+This charm simplifies initial deployment and "day N" operations of GitHub self-hosted runners. The charm makes it easy to manage self-hosted runners with security and hardware resource usage in mind.
+
+Operating your own GitHub self-hosted runner comes with [its own security concerns according to GitHub](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners#self-hosted-runner-security).
+Just like GitHub, the self-hosted runner managed by the charm are isolated in a single-use virtual machine. However, arbitrary code execution is possible under certain repositories settings. This can be utilized by malicious actors in a number of ways, such as, crypto-mining. To combat this, the charm enforce a set of GitHub repository settings to ensure code that executed in the self-hosted runner are reviewed by someone trusted.
```

Comment:
s/Just like GitHub, the self-hosted runner managed/Just like GitHub's, the self-hosted runners managed

Reply:
s/certain repositories settings/certain repository settings
s/utilized/leveraged
s/the charm enforce/the charm enforces
s/to ensure code that executed in the self-hosted runner are reviewed by someone trusted/to ensure the executed code is reviewed by someone trusted